eisbuk-admin/src/components/atoms/OrganizationSettings/OrganizationSettings.tsx

```diff
@@ -0,0 +1,396 @@
+// import { useTranslation } from "react-i18next";
+
+// import { useDispatch } from "react-redux";
+import { Formik, Field, Form, useField } from "formik";
+
+import FormControl from "@material-ui/core/FormControl";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import Cancel from "@material-ui/icons/Cancel";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { Divider, IconButton, Typography } from "@material-ui/core";
+import clsx from "clsx";
+import { useSelector } from "react-redux";
+import { getLocalAuth } from "@/store/selectors/auth";
+
+interface Props {
+  organization: OrganizationData;
+}
+const OrganizationSettings: React.FC<Props> = ({ organization }) => {
+  const classes = useStyles();
+  const userAuthInfo = useSelector(getLocalAuth);
+
+  const [enableEdit, setEnableEdit] = useState(false);
+  const handleSubmit = (orgData: OrganizationData) => {
+    console.log({ orgData });
+    // useDispatch
+    setEnableEdit(false);
+  };
+
+  const handleEnableEdit = () => {
+    setEnableEdit(!enableEdit);
+  };
+  const currentUser = userAuthInfo?.email || "";
+  return (
+    <>
+      <div className={classes.title}>
+        <Typography variant="h4">Organization Settings</Typography>
+      </div>
+      <div className={classes.content}>
+        {/* <Typography variant="h4">Organization Settings</Typography> */}
+
+        <Formik
+          {...{ initialValues: { ...organization } }}
+          onSubmit={handleSubmit}
+          validateOnChange={false}
+        >
+          {({ errors, isSubmitting, isValidating }) => (
+            <Form className={classes.form}>
+              <FormControl component="fieldset">
+                <h5 className={classes.intervalsTitle}>Admins</h5>
+                <AdminsField
+                  enableEdit={enableEdit}
+                  currentUser={currentUser}
+                />
+                <Divider />
+                <h5 className={classes.intervalsTitle}>SMTP</h5>
+
+                <div className={classes.fieldSection}>
+                  <Field
+                    label="SMTP Server"
+                    name="SMTPServer"
+                    className={classes.field}
+                    as={TextField}
+                    aria-label="SMTPServer"
+                    placeholder="SMTP server here"
+                    variant="outlined"
+                    value={organization.SMTPServer}
+                    disabled={!enableEdit}
+                  />
+                  <Field
+                    label="SMTP Port"
+                    name="SMTPPort"
+                    className={classes.field}
+                    as={TextField}
+                    aria-label="SMTPPort"
+                    placeholder="SMTP port here"
+                    variant="outlined"
+                    value={organization.SMTPPort}
+                    disabled={!enableEdit}
+                  />
```

Comment:
There is a lot of repetition (=unnecessary code) here. I see the way to reduce code and improve readability by storing each field params in an array and then render the same "template" for each field, example:
```typescript
const smtpFields: Partial<FieldProps>[] = [
  {
    // with the `name` you need to think of variable naming consistenty and conventions: use camelCase
    name: "smtpHost", 
    label: "SMTP Host"
  },
  {
    name: "smtpPort"
    label: "SMPT Port"
  }
]

return smtpFields.map((field) => (
  <
    {...field} // name and label
    // standard values shared across all five fields
    className={classes.field}
    as={TextField}
    aria-label={field.label}
    placeholder={field.label} // This might be unnecessary, if not you can just add `placeholder` in smptFields structure
    variant="outlined"
    // value={organization.SMTPPort} value is unnecessary for reasons explained above
    disabled={!enableEdit}
  />
)
```